app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,31 @@
 <resources>
     <string name="app_name">qveex</string>
+    <string name="blog_photo_content_description">Blog photo</string>
+    <string name="food_photo_content_description">Food photo</string>
+    <string name="fun_photo_content_description">Fun photo</string>
+    <string name="kid_fun_photo_content_description">Kid fun photo</string>
+    <string name="room_photo_content_description">Room photo</string>
```

Comment:
This string file is for Russian localization.

Reply:
I understand that these content descriptions were in eng lang in my code, but if you start localizing strings it must be localizied too